akka-actor/src/main/scala/akka/io/dns/DnsSettings.scala

```diff
@@ -44,6 +46,50 @@ private[dns] final class DnsSettings(system: ExtendedActorSystem, c: Config) {
 
   val ResolveTimeout: FiniteDuration = c.getDuration("resolve-timeout").asScala
 
+  private lazy val resolvConf: Option[ResolvConf] = {
+    val etcResolvConf = new File("/etc/resolv.conf")
+    // Avoid doing the check on Windows, no point
```

Comment:
Any point in opting out if this is conditional on `/etc/resolv.conf` existing?

Reply:
What if it did exist on Windows? Would using it be a good idea? That's actually the real reason, if it did exist, reading it would probably not be the right thing to do.

Reply:
I would expect nothing to exist under `/` on Windows.  And if it did exist (for whatever reason), I'd expect it using it would be the right thing.